### ISSUE TITLE
no gradient during training?

### ISSUE THREAD

**Issue by guxd** [nayeon7lee/bert-summarization#7]
In the `train_one_batch` function, you call the `self.generate_refinement_output`. 
This function has `torch.no_grad()` in side. Does that mean you do not track gradients during training?
